bin/command-line-definitions.ts

```diff
@@ -93,6 +93,14 @@ export const argDefinitions: Array<CommandLineDefinition> = [
         defaultValue: false,
         description: 'Use dump_syms to generate and upload sym files for specified binaries.',
         typeLabel: '{underline boolean} (optional)',
+    },
+    {
+        name: 'localPath',
```

Comment:
```suggestion
        name: 'local-path',
```

Reply:
We already use dumpSyms, I agree it should have been dump-syms, but I think for consistencies sake I'm going to leave this one as is. We can change them both next major version bump.